api/controllers/v1/UserController.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * UserController
+ *
+ * @description :: Server-side logic for managing users
+ * @help        :: See http://sailsjs.org/#!/documentation/concepts/Controllers
+ */
+
+module.exports = {
+
+  /**
```

Comment:
The tests are generating:
```
error: Ignored attempt to bind route (/api/v1/user) to unknown controller.action :: v1/user.index
```
Does this mean there needs to be an `index: function() "method" defined in User controller (`UserController.js)`?